src/lab6-tests.md

```diff
@@ -69,6 +69,51 @@ Unknown syntaxes are ignored.
     >>> lab6.CSSParser("foo { bar }").parse()
     [(TagSelector(tag=foo, priority=1), {})]
 
+Whitespace can be present anywhere. This is an easy mistake to make
+with a scannerless parser like used here:
+
+    >>> lab6.CSSParser("a {}").parse()
+    [(TagSelector(tag=a, priority=1), {})]
+    >>> lab6.CSSParser("a{}").parse()
+    [(TagSelector(tag=a, priority=1), {})]
+    >>> lab6.CSSParser("a{ }").parse()
+    [(TagSelector(tag=a, priority=1), {})]
+    >>> lab6.CSSParser("a {} ").parse()
+    [(TagSelector(tag=a, priority=1), {})]
+    >>> lab6.CSSParser("a {p:v} ").parse()
+    [(TagSelector(tag=a, priority=1), {'p': 'v'})]
+    >>> lab6.CSSParser("a {p :v} ").parse()
+    [(TagSelector(tag=a, priority=1), {'p': 'v'})]
+    >>> lab6.CSSParser("a { p:v} ").parse()
+    [(TagSelector(tag=a, priority=1), {'p': 'v'})]
+    >>> lab6.CSSParser("a {p: v} ").parse()
+    [(TagSelector(tag=a, priority=1), {'p': 'v'})]
+    >>> lab6.CSSParser("a {p:v } ").parse()
+    [(TagSelector(tag=a, priority=1), {'p': 'v'})]
+
+Invalid syntax is mostly skipped:
```

Comment:
Move these up under the existing "Unknown syntaxes are ignored" sentence.

Reply:
Done, don't know how I missed it. That test is already in the ones I added, so I deduped.